release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -709,6 +709,13 @@ AWS Route53 endpoint configuration is now supported on the Ingress Operator.
 
 See xref:../installing/installing_aws/installing-aws-account.html#nw-endpoint-route53_installing-aws-account[Ingress Operator endpoint configuration for AWS Route53] for more information.
 
+[id="ocp-4-6-ingress-controller-unique-id-header"]
+==== Ingress Controller unique-id configuration
+
+Configuring an Ingress Controller to inject an HTTP header with a uniquely defined request ID is now supported. This can be used to trace cluster traffic.
+
+For more information, see xref:../rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.adoc#specification[IngressController Specification]
```

Comment:
* Missing a period.
* I'd put "IngressController" in backticks, and lowercase "specification". And probably a "the " in front of the link.

Reply:
Fixed!